fission/src/ui/panels/configuring/assembly-config/interfaces/ConfigureGamepiecePickupInterface.tsx

```diff
@@ -217,8 +221,8 @@ const ConfigureGamepiecePickupInterface: React.FC<ConfigPickupProps> = ({ select
                         const robotTransformation = JoltMat44_ThreeMatrix4(
                             World.PhysicsSystem.GetBody(selectedRobot.GetRootNodeId()!).GetWorldTransform()
                         )
-                        transformGizmo.mesh.position.setFromMatrixPosition(robotTransformation)
-                        transformGizmo.mesh.rotation.setFromRotationMatrix(robotTransformation)
+                        transformGizmo.obj.position.setFromMatrixPosition(robotTransformation)
+                        transformGizmo.obj.rotation.setFromRotationMatrix(robotTransformation)
```

Comment:
Why not build a function inside of GizmoSceneObject to set these? Simply pass in either a matrix or the individual components (translation, rotation, scale) and go from there. Same with getters.